src/main/resources/config/liquibase/changelog/20190530163845_changelog.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="radchuk" id="1559227133834-1">
+        <addColumn tableName="complaint">
+            <column name="complaint_type" type="varchar(15)"/>
+        </addColumn>
+        <addNotNullConstraint columnDataType="varchar(15)"
```

Comment:
how do you make sure that for existing complaints the type is set correctly?

Reply:
I guess we need a SQL script here, that sets the type for all previously used complaints, i.e. setting the type to `COMPLAINT` for alle entries where it is null, after the column was created

Reply:
I guess you can use something like in the liquibase file

`<sql>update complaint set complaint_type = 'COMPLAINT' where complaint_type is null;</sql>`

Reply:
I know that you have the defaultNullValue below, but I would still prefer that we have consistent database entries!

Reply:
This is what `addNotNullConstraint` does already https://www.liquibase.org/documentation/changes/add_not_null_constraint.html

Reply:
ok, then it should be fine